tools/schemas/replacements-schema.json

```
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

```suggestion
  "$schema": "https://json-schema.org/draft-04/schema#",
```

Does the host support HTTPS? If yes, we should default to using HTTPS.